index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <link rel="stylesheet" href="style.css">
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+
+<body>
+  <section class="intro">
+    <header>
+      <nav>
+        <img src="images/MyLogo.png" alt="Logo">
+        <p>more options</p>
```

Comment:
- [ ] The original Figma design only has a hamburger button on the right. Kindly make changes sticking to the original design. :point_down: 
![image](https://user-images.githubusercontent.com/64894927/121442508-7f1ecd80-c994-11eb-9277-ba07a42ad21c.png)